src/modules/staking-proxy/services/staking.proxy.abi.service.ts

```diff
@@ -66,6 +78,18 @@ export class StakingProxyAbiService
         return response.firstValue.valueOf().toString();
     }
 
+    async getAllStakingFarmAddresses(
+        stakingProxyAddresses: string[],
+    ): Promise<string[]> {
+        return await getAllKeys<string>(
+            this.cachingService,
+            stakingProxyAddresses,
+            'stakeProxy.stakingFarmAddress',
```

Comment:
i would recommend putting all the keys in a `cache.info.ts` because:
- we can easily see all the caching keys in a single place
- we can easily reuse the keys somewhere else
- it helps the overall code by avoiding magic strings in the code